introduction.md

```diff
@@ -63,7 +63,9 @@ Use [our feedback portal](https://feedback.telerik.com/blazor) to tell us which
 
 ## Trial Version and Commercial License
 
-UI for Blazor is a commercial UI library. You are welcome to explore its full functionality and get technical support from the team when you register for a free 30-day trial. To use it commercially, you need to <a href="https://www.telerik.com/purchase/blazor-ui" target="_blank">purchase a license</a>. Feel free to review the Telerik UI for Blazor <a href="https://www.telerik.com/purchase/license-agreement/blazor-ui" target="_blank">License Agreement</a> to get acquainted with the full terms of use.
+UI for Blazor is a commercial UI library. You are welcome to explore its full functionality and get technical support from the team when you register for a free 30-day trial. After the trial period expires, components will be functional but a trial watermark and banner will appear.
```

Comment:
Consult
1.2.5. Limitations on Evaluation or Trial Licenses
of the license agreement
https://www.telerik.com/purchase/license-agreement/blazor-ui

After the trial period expires, the components should not be used. We can avoid disucssing this here if we just mention that the trial version has watermark and banner.